dev/gallery.tsx

```diff
@@ -113,16 +159,49 @@ export function Gallery() {
                     style={styles.cards}
                     className={isMobile ? "perseus-mobile" : ""}
                 >
-                    {questions.map((question, i) => (
-                        <QuestionRenderer
-                            key={i}
-                            question={question}
-                            apiOptions={{
-                                isMobile,
-                                flags: {mafs: mafsFlagsObject},
-                            }}
-                        />
-                    ))}
+                    {questions
+                        .filter((question) => {
+                            if (!search) {
+                                return true;
+                            }
+                            const widgetKey = Object.keys(question.widgets)[0];
```

Comment:
**thought**: We have some data in our example corpus where there is more than one widget in the `widgets` dict. We can fix this by doing `Object.keys(questions.widgets).some(key => { ... })` and nest this `switch` statement inside that. Otherwise we're only searching on the first (possibly non-interactive-widget) widget in each question.